<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Logger</class>
 <widget class="QWidget" name="Logger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1271</width>
    <height>887</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Encodage :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="encodingCBox">
       <item>
        <property name="text">
         <string>UTF-8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ASCII</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>HEX</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Ajouter Ã  la fin :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="crlfCBox">
       <item>
        <property name="text">
         <string>Pas de CR+LF</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AUTO CR+LF</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AUTO CR</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AUTO LF</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Sauvegarder</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="eraseButton">
       <property name="text">
        <string>Effacer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="logBrowser"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
